config/locales/en.yml

```diff
@@ -43,6 +43,8 @@ en:
         unit: "£"
   service_name: "Claim additional payments for teaching"
   support_email_address: "additionalteachingpayment@digital.education.gov.uk"
+  information_provided:
+    student_loan_repayment_line: "which is deducted from your claim payment if applicable"
```

Comment:
should this also be updated to include the ` and credited to your Student Loans Account` bit?

Reply:
Yeah you're right, the lines are so similar I've now extracted out an `award_description` translation which is just `£2,000 payment` or `claim payment`